cubeviz/controls/slice.py

```diff
@@ -272,6 +307,17 @@ def specviz_wavelength_slider_change(self, event=None, pos=None):
             deactivate_flag = True
             self._slider_flag = True
 
+        # The "pos" value coming from specviz appears to be related to the
+        # index in the observed wavelength and so if there is a redshift
+        # then we need to convert the pos to the rest wavelength position.
+        if self._cv_layout._units_controller and not self._cv_layout._units_controller.redshift_z == 0.0:
+            #rest_wavelength = self._cv_layout._units_controller._original_wavelengths[pos] / (1 + self._cv_layout._units_controller.redshift_z)
```

Comment:
Is there any reason to keep this line around?

Reply:
Removed